integration-tests/rita.py

```diff
@@ -323,6 +375,98 @@ def test_reach_all(self, verbose=True, global_fail=True):
                     return False
         return True
 
+    def test_routes(self, verbose=True, global_fail=True):
+        """
+        Check the presence of all optimal routes.
+        """
+        result = True
+        nodes = self.nodes
+
+        # Caution: all_routes directly relies on the layout of the netlab mesh.
+        #
+        # The routes are organized into a dictionary with nodes as keys and
+        # the expected routes as values:
+        # all_routes = {
+        #     <where_from>: [
+        #                 (<where_to>, <price>, <next_hop>),
+        #                 [...]
+        #             ],
+        #     [...]
+        # }
+
+        all_routes = {
+                self.nodes[1]: [
```

Comment:
I think this shouldn't be self to be consistent with the rest of the dictionary

Reply:
True, thanks!